feature/mpls/otg_tests/static_bgp_nexthop/static_bgp_nexthop_test.go

```diff
@@ -0,0 +1,473 @@
+	eth := flow.Packet().Add().Ethernet()
+	eth.Src().SetValue(atePort1.MAC)
+	eth.Dst().SetValue(macAddress)
+
+	mpls := flow.Packet().Add().Mpls()
+	mpls.Label().SetValue(mplsLabelV4)
+
+	ip4 := flow.Packet().Add().Ipv4()
+	ip4.Src().SetValue(atePort1.IPv4)
+	ip4.Dst().SetValue(bgpNHv4)
+
+	flow.Size().SetFixed(trafficSize)
+	flow.Rate().SetPps(trafficPps)
+}
+
+// Add IPv6 MPLS Forwarding Flow
+func iPv6MPLSForwarding(t *testing.T, config gosnappi.Config, flowname string) {
+	dut := ondatra.DUT(t, "dut")
+	macAddress := gnmi.Get(t, dut, gnmi.OC().Interface(dut.Port(t, "port1").Name()).Ethernet().MacAddress().State())
+	flow := config.Flows().Add()
+	flow.SetName(flowname)
+	flow.Metrics().SetEnable(true)
+	flow.TxRx().Port().SetTxName(config.Ports().Items()[0].Name()).SetRxNames([]string{config.Ports().Items()[1].Name()})
+	eth := flow.Packet().Add().Ethernet()
+	eth.Src().SetValue(atePort1.MAC)
+	eth.Dst().SetValue(macAddress)
+	mpls := flow.Packet().Add().Mpls()
+	mpls.Label().SetValue(mplsLabelV6)
+	ip6 := flow.Packet().Add().Ipv6()
+	ip6.Src().SetValue(atePort1.IPv6)
+	ip6.Dst().SetValue(bgpNHv6)
+	ip6.Version().SetValue(4)
+	flow.Size().SetFixed(trafficSize)
+	flow.Rate().SetPps(trafficPps)
+}
+
+// Verify Ipv4 Traffic Discard
+func verifyIpv4TrafficDiscard(t *testing.T, ate *ondatra.ATEDevice, config gosnappi.Config) {
+	ate.OTG().StartTraffic(t)
+	time.Sleep(trafficDuration * time.Second)
+	ate.OTG().StopTraffic(t)
+	if verifyFlowTraffic(t, ate, config, ipv4Flow) {
+		t.Error("Receiving IPv4 packets, RX should be zero")
+	} else {
+		t.Log("IPv4 packets dropped, Test Passed")
+	}
+}
+
+// Verify Ipv6 Traffic Discard
+func verifyIpv6TrafficDiscard(t *testing.T, ate *ondatra.ATEDevice, config gosnappi.Config) {
+	ate.OTG().StartTraffic(t)
+	time.Sleep(trafficDuration * time.Second)
+	ate.OTG().StopTraffic(t)
+	if verifyFlowTraffic(t, ate, config, ipv6Flow) {
+		t.Error("Receiving IPv6 packets, RX should be zero")
+	} else {
+		t.Log("IPv6 packets dropped, Test Passed")
+	}
+}
+
+func verifyFlowTraffic(t *testing.T, ate *ondatra.ATEDevice, config gosnappi.Config, flowName string) bool {
+	t.Log("Verify Flow Traffic")
+	startTime := time.Now()
+	count := 0
+	var got float64
+	for time.Since(startTime) < timeout {
+
+		otgutils.LogFlowMetrics(t, ate.OTG(), config)
+		countersPath := gnmi.OTG().Flow(flowName).Counters()
+		framesRx := gnmi.Get(t, ate.OTG(), countersPath.InPkts().State())
+		framesTx := gnmi.Get(t, ate.OTG(), countersPath.OutPkts().State())
+
+		if got = (math.Abs(float64(framesTx)-float64(framesRx)) * 100) / float64(framesTx); got <= lossTolerance {
+			return true
+		} else {
+			time.Sleep(interval)
+			count += 1
+		}
+	}
+
+	if count >= 2 {
+		t.Logf("Packet loss percentage for flow: got %v, want %v", got, lossTolerance)
+		return false
+	}
+	return true
+}
+
+// Enable MPLS Forwarding
+func configureMplsStaticLsps(t *testing.T, dut *ondatra.DUTDevice) {
```

Comment:
please coordinate with https://github.com/openconfig/featureprofiles/pull/4212/files#diff-b5c7bd119da0491c8203a39ae9c42b48450a258a081d59deada69e0672677469R1347

One of you should create a cfgplugin for creating static LSP's and use the same deviation.  Please avoid duplicating config generation for MPLS static LSP and associated deviations.  Thanks!

Reply:
Added in the https://github.com/openconfig/featureprofiles/pull/4139. Thanks!